src/wms/src/test/java/org/geoserver/wms/capabilities/GetCapabilitiesTransformerTest.java

```diff
@@ -415,6 +431,89 @@ private void checkVendorSpecificCapsProviders(GetCapabilitiesTransformer tr) thr
         assertEquals(1, list.getLength());
     }
 
+    /**
+     * Gets a capabilities document as an XML string Also, adds: <!DOCTYPE WMT_MS_Capabilities
+     * SYSTEM "WMS_MS_Capabilities.dtd"> at the top of the document.
+     */
+    String getCapabilitiesXML() throws Exception {
+        // WMS wms = new WMS(getGeoServer());
+        WMS wms = (WMS) applicationContext.getBean("wms");
```

Comment:
This is the only place I can see where spring would be unhappy...  I copied this from another test...